packages/mobile/src/pincode/PincodeSet.tsx

```diff
@@ -104,6 +107,9 @@ export class PincodeSet extends React.Component<Props, State> {
       this.props.setPincode(PincodeType.CustomPin)
       ValoraAnalytics.track(OnboardingEvents.pin_set)
       this.navigateToNextScreen()
+      if (this.props.route.params.changePin) {
+        Logger.showMessage('PIN changed')
```

Comment:
This should still be translated :)

Reply:
using the i18n library?

Reply:
waiting for the right translations from the slack channels :3

Reply:
No need to translate manually, sorry my comment wasn't clear 😅  We just need to put the string in the locale file in English and the product team will take care of translating to other languages with Crowdin